packages/pageheader/components/PageHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+import { default as Breadcrumb } from "../../breadcrumb/components/Breadcrumb";
+
+import { flush } from "../../shared/styles/styleUtils";
+
+import { style } from "../style";
+
+export interface PageHeaderProps {
+  breadcrumbElements: React.ReactNodeArray;
```

Comment:
Given the way that it is used, this could be a ReactNode, which is a superset containing ReactNodeArray

Reply:
I will check this to make sure the type is preventing what I wanted to prevent with it. Thanks for mentioning this. So the type `ReactNodeArray` is not equal to `ReactNode[]` ? That was what I wanted to achieve.